### PR TITLE
[OSDEV-911] Add code quality pipelines for dedupe-hub | Main Part

### PR DESCRIPTION
**Pull request by roman-stolar**
[OSDEV-911](https://opensupplyhub.atlassian.net/browse/OSDEV-911) Add code quality pipelines for dedupe-hub | Main Part

- Created tests config + added simple test
- Updated code_quality pipeline

### CODE REVIEW

**Comment by roman-stolar** [comment]
Ignore code coverage result for Dedupe Hub, as base/main branch didn't have any tests yet.

**Review by killev — APPROVED** [review]

**Review by VadimKovalenkoSNF — DISMISSED** [review]
Approved

**Review by killev — APPROVED** [review]

**Review by VadimKovalenkoSNF — APPROVED** [review]
Approved

File: src/dedupe-hub/api/Dockerfile [hunk 1]
```diff
@@ -7,3 +7,4 @@ COPY ./requirements.txt ./requirements.txt
 RUN pip install --no-cache-dir --upgrade -r requirements.txt
  
 COPY ./app ./app
+COPY ./tests ./tests
```
Comment (killev) [inline src/dedupe-hub/api/Dockerfile:10]: It isn't good to have tests in production... Still they go to the image everywhere including Django. So I think it worth creating ticket regarding this.

Comment (roman-stolar) [reply src/dedupe-hub/api/Dockerfile:10]: Yes, we have same situation on Django, need to think how to hide it for production & create a ticket for it.

**Pull request merged**